<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fMainWindow</class>
 <widget class="QMainWindow" name="fMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manabu</string>
  </property>
  <widget class="QWidget" name="fCentralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="6">
     <widget class="QTreeWidget" name="fDeckSelector">
      <property name="minimumSize">
       <size>
        <width>313</width>
        <height>0</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string>Decks</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="fAnswer"/>
    </item>
    <item row="5" column="5">
     <widget class="QComboBox" name="fDeckNumber"/>
    </item>
    <item row="2" column="2" colspan="3">
     <widget class="QLabel" name="fResult">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="fWord">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="fMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="fMenuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="fActionOpen"/>
    <addaction name="fActionSaveDeck"/>
    <addaction name="fActionExit"/>
   </widget>
   <widget class="QMenu" name="fMenuDeck">
    <property name="title">
     <string>Deck</string>
    </property>
    <addaction name="fActionToggleDisplay"/>
   </widget>
   <addaction name="fMenuFile"/>
   <addaction name="fMenuDeck"/>
  </widget>
  <widget class="QStatusBar" name="fStatusBar"/>
  <action name="fActionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="fActionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="fActionSaveDeck">
   <property name="text">
    <string>Save Deck</string>
   </property>
  </action>
  <action name="fActionToggleDisplay">
   <property name="text">
    <string>Display Back</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
